cookbook/how_to_apply_patch.rst

```diff
@@ -0,0 +1,123 @@
+.. index::
+    single: Patch
+
+How to apply patch
+==================
+
+The patch file name supports the following naming convention: {package}-{version_to_apply}.patch.
+For example, platform-1.9.2.patch.
+
+**1** Create backups of your Database and Code.
+
+**2** "Cd" to the crm root folder and switch the application to the maintenance mode.
+
+.. code-block:: bash
+
+    $ cd /path/to/application
+    $ sudo -uwww-data app/console lexik:maintenance:lock --env prod
+
+
+**3** Stop the cron tasks
+
+.. code-block:: bash
+
+    $ crontab -e -uwww-data
+
+    .. note::
+        www-data can be changed to user under which your web server runs
+
+Comment this line.
+
+.. code-block:: text
+
+     */1 * * * * /usr/bin/php /path/to/application/app/console --env=prod oro:cron >> /dev/null
+
+Kill the related job daemon process.
+
+.. code-block:: bash
+
+    $ ps ax|grep php5
+    $ kill <process_pid>
+
+<process_pid> - is a PID of currently executing application job daemon process. For example:
+
+.. code-block:: text
+
+    /path/to/application/app/console jms-job-queue:run --max-runtime=3600 --max-concurrent-jobs=5 --env=prod
+
+**4** Copy the patch file to the package directory
+
+.. code-block:: text
+
+/path/to/application/vendor/oro/{package}
+
+So, "platform-1.9.2.patch" file should be copied to "/path/to/crm_folder/vendor/oro/platform".
+
+**5** "cd" to the package folder and execute "patch" command to apply the patch.
```

Comment:
Edit:  To apply the patch, "cd" to the package folder and execute "patch" command.